client/my-sites/site-settings/composing/publish-confirmation.jsx

```diff
@@ -55,8 +55,8 @@ class PublishConfirmation extends Component {
 
 				<FormSettingExplanation isIndented>
 					{ translate(
-						'This adds a confirmation step with helpful settings and tips for double' +
-						'checking your content before publishing.'
+						'This adds a confirmation step with helpful settings and tips ' +
```

Comment:
Hi! I've found a possible matching string that has already been translated **2** times:
`translate( 'This adds a confirmation step with helpful settings and tips for doublechecking your content before publishing.' )` *ES Score: 11*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).